portal/v2/src/Request.tsx

```diff
@@ -125,3 +126,16 @@ export const RequestKubeconfig = async (
     return OnError(err)
   }
 }
+
+export const FetchStatistics = async (cluster: ICluster, statisticsName: string, duration: string, endDate: Date): Promise<AxiosResponse | null> => {
```

Comment:
Variable name shouldn't start with an upper-case letter as it's not a component.

Reply:
it is actually the fetch function and it was as is from the starting for I continued with the same style.

Reply:
I understand, however ideally the code should be following known React conventions, so while we're here touching this code, it would be better to make it right rather than leaving it for the next person to have to clean it up.

Reply:
I have changed all the functions in that file.